GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -326,6 +326,14 @@ public FormBrowse(GitUICommands commands, string filter, ObjectId? selectedId =
 
             RevisionGrid.ToggledBetweenArtificialAndHeadCommits += (s, e) => FocusRevisionDiffFileStatusList();
 
+            toolPanel.TopToolStripPanel.BackColor = Color.Transparent;
+            mainMenuStrip.BackColor = Color.Transparent;
+            ToolStripMain.BackColor = Color.Transparent;
+            ToolStripFilters.BackColor = Color.Transparent;
+            ToolStripScripts.BackColor = Color.Transparent;
+
+            BackColor = Color.FromArgb(251, 251, 251).AdaptBackColor();
```

Comment:
```
BackColor = Color.FromArgb(251, 251, 251).AdaptBackColor();
// ...
var panelBorderColor = Color.FromArgb(224, 224, 224).AdaptBackColor()
```
Since `.AdaptBackColor()` and similar methods were introduced, they lead to such in-place color definitions here and there, which makes me want for a more centralized registry of colors used in the application.

Maybe a static class, somewhere near `AppColorDefaults`.

Then it's easier to find an already existing color suitable for the task in hand and more difficult to accidentally introduce a duplicate.

Reply:
Moved to OtherColors class for now.